### PR TITLE
Adding european Data-Center

### PR DESCRIPTION
**Pull request by svwu**
Hi there!
As we are using the europea data center, we also need to use that specific API, as the "standard" saucelabs.com API doesn't work for us.

I've changed every parts to defaultly use saucelabs.com and whenever you need to, you can switch to the european version.

Best,
Sven

### CODE REVIEW

File: src/Sauce/Sausage/SauceConfig.php [hunk 1]
```diff
@@ -75,8 +83,8 @@ public static function GetBuild()
         return false;
     }
 
-    public static function WriteConfig($username, $access_key) {
-        file_put_contents(CONFIG_PATH, "{$username},{$access_key}");
+    public static function WriteConfig($username, $access_key, $sauce_host = 'saucelabs.com') {
+        file_put_contents(CONFIG_PATH, "{$username},{$access_key},$sauce_host");
```
Comment (jlipps) [inline src/Sauce/Sausage/SauceConfig.php]: any reason not to follow the same pattern as the other variables?

**Review by jlipps — APPROVED** [review]

File: src/Sauce/Sausage/SauceConfig.php (hunk 1 above)
Comment (svwu) [reply src/Sauce/Sausage/SauceConfig.php]: I tried to keep the variable pattern as the others.
Regulary I use camelCase, instead, so I might have missed some?

Comment (jlipps) [reply src/Sauce/Sausage/SauceConfig.php]: I mean that the others have braces around them and the new one doesn't. Is there a reason?

Comment (svwu) [reply src/Sauce/Sausage/SauceConfig.php]: Ah, this is inconsistent, I will change that!

**Review by jlipps — APPROVED** [review]

**Comment by jlipps** [comment]
OK, it's been tagged as 0.18.0. I think packagist should pick this up and build a release from it.

**Comment by svwu** [comment]
Somehow packagist didn't care for the new version can you manually trigger it?

**Comment by jlipps** [comment]
OK, I think I have done that.

**Pull request merged**